src/common/low_precision_transformations/src/fold_fake_quantize.cpp

```diff
@@ -28,39 +32,85 @@ FoldFakeQuantizeTransformation::FoldFakeQuantizeTransformation(const Params& par
         return transform(*context, m);
     };
 
-    auto m = std::make_shared<ngraph::pattern::Matcher>(fakeQuantize, matcher_name);
+    auto m = std::make_shared<ngraph::pattern::Matcher>(matcher, matcher_name);
     this->register_matcher(m, callback);
 }
 
 bool FoldFakeQuantizeTransformation::transform(TransformationContext& context, ngraph::pattern::Matcher &m) {
     const auto fakeQuantize = ov::as_type_ptr<opset1::FakeQuantize>(m.get_match_root());
-    if (fakeQuantize == nullptr) {
-        return false;
+    if (fakeQuantize) {
+        if (!canBeTransformed(context, fakeQuantize)) {
+            return false;
+        }
+
+        const auto constantShape = fakeQuantize->input(1).get_partial_shape();
+        if (constantShape.is_dynamic()) {
+            return false;
+        }
+
+        std::shared_ptr<ngraph::Node> resultConstant = NetworkHelper::fold_fake_quantize(
+            fakeQuantize,
+            false,
+            ((constantShape.rank().get_length() >= 2) && (constantShape[1] != 1ul)) ? 1ul : 0ul);
+        if (ov::is_type<opset1::Constant>(resultConstant)) {
+            replace_node(fakeQuantize, resultConstant);
+            return true;
+        }
     }
 
-    if (!canBeTransformed(context, fakeQuantize)) {
-        return false;
+    //    y = Multiply(x, [0,0,....,0])
+    // => y = Broadcast(Constant(0, typeof(y))), ShapeOf(x))
+    const auto dequantMultiply = ov::as_type_ptr<opset1::Multiply>(m.get_match_root());
+    if (dequantMultiply) {
+        auto multiplyInputs = dequantMultiply->input_values();
+        auto constScales = as_type_ptr<opset1::Constant>(multiplyInputs[0].get_node_shared_ptr());
+        auto dataNode = multiplyInputs[1].get_node_shared_ptr();
+        if (!constScales) {
+            dataNode = multiplyInputs[0].get_node_shared_ptr();
+            constScales = as_type_ptr<opset1::Constant>(multiplyInputs[1].get_node_shared_ptr());
+        }
```

Comment:
I suggest using `NetworkHelper::getConstantInput` here to get `constScales`

Reply:
Sorry, but I also needs dataNode which we cannot deduce easily from `NetworkHelper::getConstantInput`'s return value, thus I prefer to leave it as it is